content/en/networks/mainnet/network-performance/index.md

```diff
@@ -27,16 +27,16 @@ It is highly non-trivial to provide highly reliable benchmarks for Filecoin netw
 A message that requires [transferring FIL](https://lotus.filecoin.io/docs/set-up/manage-fil/) is often extremely fast, and will take on average ~1 blocktime (or around 30 seconds) to be reflected on-chain. We consider 120 blocks (1 hour) a conservative number of confirmations for high-value transfers.
 
 ## Data storage
-
+<!-- markdown-link-check-disable -->
 In the Filecoin [data storage protocol](https://lotus.filecoin.io/docs/developers/store-data/), the following occurs once a deal is proposed and accepted:
-
+<!-- markdown-link-check-enable -->
```

Comment:
Link checker flagged https://lotus.filecoin.io/docs/developers/store-data/. Manual check confirmed that this page has been moved. Not sure where it should actually go. Disabled link check to pass pre-commit until this is figured out and edited accordingly.

@johnnymatthews